test/sourcemaps.js

```diff
@@ -61,6 +61,35 @@ describe('sourcemaps', function() {
     assert.equal(output.env(), process.env.NODE_ENV);
   });
 
+  it('should create a valid sourcemap as a child of a nested TS bundle', async function() {
+    let b = await bundle(
+      __dirname + '/integration/sourcemap-typescript-nested/index.ts'
+    );
+
+    assertBundleTree(b, {
+      name: 'index.js',
+      assets: ['index.ts', 'local.ts'],
+      childBundles: [
+        {
+          name: 'index.map',
+          type: 'map'
+        }
+      ]
+    });
+
+    let raw = fs
+      .readFileSync(path.join(__dirname, '/dist/index.js'))
+      .toString();
+    let map = fs
+      .readFileSync(path.join(__dirname, '/dist/index.map'))
+      .toString();
+    mapValidator(raw, map);
+
+    let output = run(b);
+    assert.equal(typeof output.env, 'function');
+    assert.equal(output.env(), process.env.NODE_ENV);
+  });
```

Comment:
This test seemed to pass even without the fix for me... How can we reproduce the issue?

Reply:
I was unfortunately not able to reproduce the issue, I wrote this test in the hopes it would fail. It didn't and i just left it in, because of better code coverage.
Maybe @brandon93s or @robcrocombe can provide us with a better test?